token-lending/program/Cargo.toml

```diff
@@ -16,19 +16,19 @@ arrayref = "0.3.6"
 bytemuck = "1.5.1"
 num-derive = "0.3"
 num-traits = "0.2"
-solana-program = "1.7.12"
-spl-token = { path = "../../token/program", features = [ "no-entrypoint" ] }
-switchboard-program = "0.1.45"
+solana-program = "=1.7.12"
+spl-token = { version = "3.1.0", features=["no-entrypoint"] }
```

Comment:
3.3.0 ?

Reply:
um yea sure can do ig
was messing around with the spl-token version to figure out what was the breaking change